packages/react/src/components/PageHeader/PageHeader.tsx

```diff
@@ -317,13 +318,39 @@ const PageHeaderTabBar = React.forwardRef<
   );
   return (
     <div className={classNames} ref={ref} {...other}>
-      <p>page header tab bar</p>
       {children}
     </div>
   );
 });
 PageHeaderTabBar.displayName = 'PageHeaderTabBar';
 
+interface PageHeaderTabsProps extends React.ComponentProps<typeof Tabs> {
+  children?: React.ReactNode;
+  className?: string;
+}
+
+const PageHeaderTabs = React.forwardRef<HTMLDivElement, PageHeaderTabsProps>(
+  function PageHeaderTabs(
+    { className, children, ...other }: PageHeaderTabsProps,
+    ref
+  ) {
+    const prefix = usePrefix();
+    const classNames = classnames(
+      {
+        [`${prefix}--page-header__tab-bar__tabs`]: true,
+      },
+      className
+    );
+
+    return (
+      <div className={classNames} ref={ref}>
+        <Tabs {...other}>{children}</Tabs>
+      </div>
```

Comment:
Is this `div` wrapper needed right now? Could we spread the `classNames` and `ref` directly onto `Tabs` instead? That would make this component's API more transparent towards `Tabs` usage.